database/src/main/scala/cromwell/database/migration/restart/table/RenameWorkflowOptionKeysMigration.scala

```diff
@@ -0,0 +1,61 @@
+package cromwell.database.migration.restart.table
+
+import cromwell.database.migration.restart.table.RenameWorkflowOptionKeysMigration._
+import liquibase.database.jvm.JdbcConnection
+import spray.json._
+
+
+object RenameWorkflowOptionKeysMigration {
+  private val RenamedOptionKeys = Map(
+    "defaultRuntimeOptions" -> "default_runtime_attributes",
+    "workflowFailureMode" -> "workflow_failure_mode",
+    "workflow_log_dir" -> "final_workflow_log_dir",
+    "outputs_path" -> "final_workflow_outputs_dir",
+    "call_logs_dir" -> "final_call_logs_dir"
+  )
+
+  private val QueryWorkflowStore =
+    """
+       SELECT WORKFLOW_STORE_ID, WORKFLOW_OPTIONS FROM WORKFLOW_STORE
+    """
+
+  private val UpdateWorkflowStore =
+    """
+       UPDATE WORKFLOW_STORE SET WORKFLOW_OPTIONS = ? WHERE WORKFLOW_STORE_ID = ?
+    """
+}
+
+
```

Comment:
IMHO, there's a ton of extra whitespace above, and those aren't even really multi-line strings.